src/coreclr/tools/Common/TypeSystem/IL/ILImporter.cs

```diff
@@ -212,7 +215,13 @@ private void FindJumpTargets()
                                 CreateBasicBlock(target);
                             else
                                 ReportInvalidBranchTarget(target);
-                            CreateBasicBlock(_currentOffset);
+                            if ((uint)_currentOffset < (uint)_basicBlocks.Length)
+                                CreateBasicBlock(_currentOffset);
+                            else
+                            {
```

Comment:
Formatting: If one side of if-else has curly braces, the other side should have them too.